src/endpoint/s3/s3_rest.js

```diff
@@ -216,7 +217,7 @@ async function authorize_request_policy(req) {
         return;
     }
 
-    const account = await req.object_sdk.rpc_client.account.read_account({});
+    const account = req.object_sdk.requesting_account;
```

Comment:
@jackyalbo @romayalon FYI - this is the fix we discussed, instead of read_account, it reuses the same account_cache that object_sdk uses to check the signatures.

Reply:
by default AFM will fetch the 4M block that includes this byte, so if you need 8M we need to set the chunck size for AFM to 12M, the other option is to set the offset to the first byte of the 4M block, in that case we can set the chunck size to 8M, this assume the block size is 4M and that NooBaa is checking for the block size.